### ISSUE TITLE
Add setPlayback to iOS

### ISSUE THREAD

**Issue by Guichaguri** [tanguyantoine/react-native-music-control#36]
Currently, setPlayback is only available in Android. Would be nice to have support this function in iOS devices.
Options that can't be implemented in iOS should be ignored and a note should be included in the README clarifying what is Android-only and what is iOS-only

```js
MusicControl.setPlayback({
  state: MusicControl.STATE_PLAYING, // (STATE_ERROR, STATE_STOPPED, STATE_PLAYING, STATE_PAUSED, STATE_BUFFERING)
  volume: 10, // (Integer from 0 to maxVolume)
  maxVolume: 10, // (Integer)
  speed: 1, // Playback Rate (Double)
  elapsedTime: 103, // Elapsed Seconds (Double)
  bufferedTime: 200 // Buffered Seconds (Double)
})
```

**Comment by tanguyantoine** [comment]
* speed: playbackRate
* elapsedTime: elapsedPlaybackTime
* state: what is that referring to ?
* volume/maxVolume : looks to not be customizable in iOs

**Comment by Guichaguri** [comment]
Wouldn't it be better to move `playbackRate` and `elapsedPlaybackTime` to match the Android side? I've created the `setPlayback` function is for performance reasons, so you wouldn't have to update the metadata every time you just wanted to update the state.

`state` is whether the player is playing, paused, stopped, buffering, etc. Unfortunately, I don't think it's supported in iOS, I've only found an option for [macOS](https://developer.apple.com/reference/mediaplayer/mpnowplayingplaybackstate?language=objc)

Any option that is not customizable in iOS should be ignored
